### PR TITLE
Add core files. 

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "bedrock-vc-status-list-context",
+  "version": "1.0.0",
```
Comment (dlongley) [inline package.json]: I think this has to be less than 1.0 to allow for a major version publish at 1.0 -- but could be wrong about that.

```suggestion
  "version": "0.0.1",
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "description": "Bedrock VC Status List 2021 Context (Bedrock Module)",
```
Comment (dlongley) [inline package.json]:
```suggestion
  "description": "VC Status List 2021 Context for Bedrock apps",
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com"
+  },
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/bedrock-vc-status-list-context/issues"
+  },
+  "homepage": "https://github.com/digitalbazaar/bedrock-vc-status-list-context#readme",
+  "dependencies": {
+    "vc-status-list-context": "~1.0.0"
```
Comment (dlongley) [inline package.json]: Is this intentional -- to only accept patches and not minor upgrades?

```suggestion
    "vc-status-list-context": "^1.0.0"
```

**Review by dlongley — APPROVED** [review]
Approving, but see comments. Thanks!

File: package.json (hunk 1 above)
Comment (JSAssassin) [reply package.json]: Fixed a92f012

File: package.json (hunk 2 above)
Comment (JSAssassin) [reply package.json]: Updated a92f012

File: package.json (hunk 3 above)
Comment (JSAssassin) [reply package.json]: Fixed a92f012

**Pull request merged**